contracts/staking/PoSValidatorManager.sol

```diff
@@ -38,6 +40,8 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
             _pendingEndDelegatorMessages;
         /// @notice Maps the validationID to the uptime of the validator.
         mapping(bytes32 validationID => uint64) _validatorUptimes;
+        /// @notice Maps the validationID to the delegation fee rate.
+        mapping(bytes32 validationID => uint256) _validatorDelegationFeeRates;
```

Comment:
I do like keeping the staking specific settings in the staking contract. We should think about if this is also the simplest solution given the PoA -> PoS upgrade potential, which I think it does? If we allow Validators to set their fee rate to 0 to disable delegation, then we could automatically get the property of PoA validators not being able to be delegated to, I think?

Reply:
I don't think that I love overloading the function of the fee rate, but this might be the cheapest/cleanest way of doing this. I will give it a try and consider alternatives.